lib/waistband/index.rb

```diff
@@ -261,6 +265,18 @@ def alias_exists?(alias_name)
       )
     end
 
+    def parent_index_names
+      client.indices.get_alias(name: config_name).map { |k, v| k }
+    rescue Elasticsearch::Transport::Transport::Errors::NotFound
+      nil
```

Comment:
I'd rather return an empty array here and keep the return type consistent.

Reply:
True, much better